airmozilla/search/templates/search/home.html

```diff
@@ -92,7 +92,10 @@ <h5 class="entry-upcoming">
 
           <a class="go" href="{{ href }}">{{ _('See more') }}</a>
         </p>
-        <a class="star" data-id="{{ event.id }}" title="Save as a starred event"></a>
+        <a class="star" data-id="{{ event.id }}"
+            data-ttOn="Remove starred event"
+            data-ttOff="Save as a starred event"
+            ></a>
```

Comment:
This is now the 4th place where we write this down. 

To begin with, I think we should change the wording from "Remove starred event" to "Remove star" (because you're not removing the event, after all). But it means you'd have to do a careful search-and-replace and there's a chance you miss one possibly. 

The only difference is that they have a different extra class. 

The right thing to do would now be to move this piece of HTML into a template helper. I would suggest you create a file called `helpers.py` in the `starred` app and register a function called something like `startag()` which could look something like this:
```python
@register.function
def startag(id, extra_class=''):
    if isinstance(extra_classes, basestring):
       extra_classes = [extra_classes]
    extra_classes.insert(0, 'star')
    return '<a class="{0}" data-id="{1}" data-star-on="Remove..." data-star-off="Save..."></a>'.format(
        ' '.join(extra_classes), 
        id
    )
    
```